backend/api/src/handlers/dynamodb/get-service-clients.ts

```diff
@@ -1,17 +1,19 @@
-import {APIGatewayProxyEvent, APIGatewayProxyResult} from "aws-lambda";
 import DynamoDbClient from "../../dynamodb-client";
 
 const client = new DynamoDbClient();
 
-export const getServiceClientsHandler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
-    const serviceId = event.pathParameters?.serviceId;
-    if (serviceId === undefined) {
+export type getServiceHandlerInvokeEvent = {
```

Comment:
Sorry if this seems a but back and forth-y, but I do think this handler **is** invoked by an API gateway at the endpoint `/get-services/{userId}`, so I think we should leave this type as is. It is bit confusing because there are a lot of lambdas in our backend and it's hard to know what is state machine invoked and what is API Gateway invoked. In this case you can find the handler in the `api.template.yml` with this section: 
```
Events:
        Api:
          Type: Api
          Properties:
            RestApiId: !Ref API
            Path: /get-services/{userId}
            Method: GET
```

Reply:
Thanks: it is a fair point. Reverted this and fixed the unit test to accommodate .